yt_dlp/extractor/xhamster.py

```diff
@@ -20,7 +20,7 @@
 
 
 class XHamsterIE(InfoExtractor):
-    _DOMAINS = r'(?:xhamster\.(?:com|one|desi)|xhms\.pro|xhamster\d+\.com|xhday\.com|xhvid\.com)'
+    _DOMAINS = r'(?:xhamster\.(?:com|one|desi)|xhms\.pro|xhamster\d+\.(com|desi)|xhday\.com|xhvid\.com)'
```

Comment:
```suggestion
    _DOMAINS = r'(?:xhamster\.(?:com|one|desi)|xhms\.pro|xhamster\d+\.(?:com|desi)|xhday\.com|xhvid\.com)'
```

[**Don't capture groups you don't use**](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#dont-capture-groups-you-dont-use)